laa_court_data_api_app/routers/hearing_events.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from uuid import UUID
+
+from fastapi import APIRouter, Query
+from fastapi.responses import Response
+
+from laa_court_data_api_app.internal.court_data_adaptor_client import CourtDataAdaptorClient
+from laa_court_data_api_app.models.hearing_events.hearing_events_response import HearingEventsResponse
+from laa_court_data_api_app.models.hearing_events.hearing_events_result import HearingEventsResult
+
+logger = logging.getLogger(__name__)
+router = APIRouter()
+
+
+@router.get("/v2/hearing_events/{hearing_id}", response_model=HearingEventsResponse, status_code=200)
+async def get_hearing_events(hearing_id: UUID, date: str = Query(None, example="2001-01-20")):
+    logger.info("Calling_Hearing_Events_Get_Endpoint")
+    logging.info(f"Hearing_Events_Get_{hearing_id}_{date}")
+    client = CourtDataAdaptorClient()
+    cda_response = await client.get(f'/api/internal/v2/hearings/{hearing_id}/event_log/{date}')
+
+    if cda_response is None:
+        logging.error("Hearing_Events_Endpoint_Did_Not_Return")
+        return Response(status_code=424)
+
+    match cda_response.status_code:
+        case 200:
+            logging.info("Hearing_Events_Endpoint_Returned_Success")
+            hearing_events_result = HearingEventsResult(**cda_response.json())
+            return HearingEventsResponse(**hearing_events_result.dict())
+        case 400:
+            logging.info("Hearing_Events_Endpoint_Validation_Failed")
```

Comment:
I noticed there's inconsistency between the different routers in terms of logging to info or errors for these. It seems to make sense to log to errors rather than info in the same way we do for if cda_response is None

Reply:
done thanks